### ISSUE TITLE
Collab Cycle: Contact Center Review

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#29792]

**Comment by srsuddath** [comment]
@denisecoveyduc can we close this ticket?

**Comment by denisecoveyduc** [comment]
@srsuddath @nicole-leonard  Hi.. so this looks like a contact center review for typeahead 2.0 which happened after my departure. @LindaLi123 may have reached out to the contact center team or may not have, I am not sure. 

The contact center team had what they needed for 1.0 so unless there were major changes I would assume they have enough info. To be sure I'd recommend you reach out the contact center team just to check.

**Comment by jwoodman5** [comment]
@nicole-leonard I looked myself as did our content specialist who performs the reviews for our team. We don't recall doing a review for Typehead 2.0 and could not find any record of us doing one. 

@denisecoveyduc I may recall chatting with you at some point about whether or not a product guide review would be necessary for a particular release, but don't recall if this was it.

As Denise mentioned, however, we may not need to do anything depending on the impact of the changes. Nicole, maybe we can chat about this quickly when we meet tomorrow?

**Comment by denisecoveyduc** [comment]
@jwoodman5 You are right, we connected back in the days of typeahead 1.0 and at the time we felt that the experience didn't really warrant a full product guide since it seemed that time would be better spent on a product guide for search/navigation across va.gov.  We had thought the typeahead feature may be included in a product guide for search once it was established. That is my recollection anyways.

**Comment by jwoodman5** [comment]
Thanks @denisecoveyduc That all makes sense to me.

**Comment by jwoodman5** [comment]
@nicole-leonard @denisecoveyduc Seems like we can close this out and make sure this functionality is included with a `search` product guide. Does that sound right?
